src/components/routes/Account/FirebaseAccountEmailHandler.tsx

```diff
@@ -29,42 +37,97 @@ const DefaultContainer = styled.div`
     section {
```

Comment:
can we move this css to a container to stick with our styling architecture?

Reply:
done